pyhf/tensor/tensorflow_backend.py

```diff
@@ -157,9 +174,39 @@ def generic_len(a):
             raise error
 
         broadcast = [arg if generic_len(arg) > 1 else
-                     tf.tile(tf.slice(arg, [0], [1]), tf.stack([max_dim])) for arg in args]
+                     self.tile(tf.slice(arg, [0], [1]), tf.stack([max_dim])) for arg in args]
         return broadcast
 
+    def tile(self, A, reps):
```

Comment:
I think we can remove `tile` and thus reduce surface-area we need to sync between backends